Tasks/DownloadBuildArtifacts/package-lock.json

```diff
@@ -20,14 +20,29 @@
       "integrity": "sha1-SlKCrBZHKek2Gbz9OtFR+BfOkfU="
     },
     "artifact-engine": {
-      "version": "0.1.15",
-      "resolved": "https://registry.npmjs.org/artifact-engine/-/artifact-engine-0.1.15.tgz",
-      "integrity": "sha512-bD6lpUNoiWkTy1th7UKSjx1qQ1i2pWiqJDjVE4RT3tQznXybP8cTjsZBmwOxz633axgsGfLR2lWyKHqwU4loQQ==",
+      "version": "0.1.17",
+      "resolved": "https://registry.npmjs.org/artifact-engine/-/artifact-engine-0.1.17.tgz",
+      "integrity": "sha512-Ls7i8QpEHOmFTQrSr/jk/23bj7vbSZpuy9EJNeuvFzaE1N523wiTjXtiE1wUCV4NbrwZBPomjKvLksv/VRMesw==",
       "requires": {
         "handlebars": "4.0.10",
         "minimatch": "3.0.2",
         "tunnel": "0.0.4",
-        "vsts-task-lib": "2.1.0"
+        "vsts-task-lib": "2.2.1"
+      },
+      "dependencies": {
+        "vsts-task-lib": {
+          "version": "2.2.1",
```

Comment:
@omeshp this is auto generated, i see it picked the dependencies of artifact-engine and updated it here. This is fine I guess.